packages/tools/devtools/devtools-browser-extension/src/popup/PopupScript.ts

```diff
@@ -19,7 +23,26 @@ import { browser } from "../Globals";
 //   If not, we may want to display an error message with a link to docs explaining how to
 //   use them.
 
-browser.tabs.query({ active: true, currentWindow: true }, (tab) => {
+// TODO: double check that tab selection can't change while popup is being displayed.
+browser.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+	if (tabs.length === 0) {
+		console.debug("No active tab.");
+		return;
+	}
+
+	// TODO: verify this
```

Comment:
It would probably be good to do a very short investigation into whether or not this is ever expected to happen. We can update our logging to be an error or not depending on the documentation.

Reply:
Ideally we can remove this TODO before merging.